Firmware/ultralcd.cpp

```diff
@@ -7457,13 +7447,11 @@ void menu_lcd_longpress_func(void)
 
     // explicitely listed menus which are allowed to rise the move-z or live-adj-z functions
     // The lists are not the same for both functions, so first decide which function is to be performed
-    if ( (moves_planned() || IS_SD_PRINTING || usb_timer.running() )){ // long press as live-adj-z
-        if (( current_position[Z_AXIS] < Z_HEIGHT_HIDE_LIVE_ADJUST_MENU ) // only allow live-adj-z up to 2mm of print height
-        && ( menu_menu == lcd_status_screen // and in listed menus...
+    if (BABYSTEP_ALLOWED()){ // long press as live-adj-z
```

Comment:
oh, this is the primary spot for the `current_position[Z_AXIS] < Z_HEIGHT_HIDE_LIVE_ADJUST_MENU` check

Reply:
Github marks this as outdated due to me rebasing the branch to sync with MK3. This comment is still relevant 👍

Reply:
[e3b9e9e](https://github.com/prusa3d/Prusa-Firmware/pull/4063/commits/e3b9e9efe0aeb9361faa7a7e93c877f250c540fe) This is now applied everywhere so it's consistant in behavior.